speedy.shtml

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html class="no-js" lang="en"><!-- InstanceBegin template="/Templates/speedy.dwt" codeOutsideHTMLIsLocked="false" -->
+<head>
+    <!--#include virtual="/wdn/templates_4.1/includes/metanfavico.html" -->
+    <!--
+        Membership and regular participation in the UNL Web Developer Network is required to use the UNLedu Web Framework. Visit the WDN site at http://wdn.unl.edu/. Register for our mailing list and add your site or server to UNLwebaudit.
+        All framework code is the property of the UNL Web Developer Network. The code seen in a source code view is not, and may not be used as, a template. You may not use this code, a reverse-engineered version of this code, or its associated visual presentation in whole or in part to create a derivative work.
+        This message may not be removed from any pages based on the UNLedu Web Framework.
+    
+        $Id: speedy.dwt | 6edb0e1ee94038935f3821c6ce15dfd5c217b2e2 | Tue Dec 1 17:08:56 2015 -0600 | Kevin Abel  $
+    -->
+    <!--#include virtual="/wdn/templates_4.1/includes/scriptsandstyles_speedy.shtml" -->
```

Comment:
I advise against using an `shtml` include here. While server-side includes _can_ be nested like this, the files we put in the `includes` directory must be synced across a number of platforms and server technologies. SSI is not likely to be a technology that is widely supported and deployed.